### PR TITLE
Follow up patches to _LINKSTO

### PR DESCRIPTION
**Pull request by alistair3149**
This PR is made in reference to: #216

- Add documentation for `_LINKSTO`
- Address various issues raised by phpcs

### CODE REVIEW

**Comment by alistair3149** [comment]
Marking as WIP to address other issues raised by CI

**Comment by alistair3149** [comment]
@jaideraf ready for review

File: src/PropertyAnnotators/LinksToPropertyAnnotator.php [hunk 1]
```diff
@@ -14,15 +14,15 @@
...
+ * @license GPL-2.0-or-later
  * @since 3.0.4
  */
 class LinksToPropertyAnnotator implements PropertyAnnotator {
 
 	/**
 	 * Predefined property ID
 	 */
-	const PROP_ID = '___LINKSTO';
+	public const PROP_ID = '___LINKSTO';
```
Comment (JeroenDeDauw) [inline src/PropertyAnnotators/LinksToPropertyAnnotator.php:25]: This requires PHP 8.1. SMW is keeping compat with 8.0, and this extension still has tests for 7.4, though I suppose those could be removed.

Comment (alistair3149) [reply src/PropertyAnnotators/LinksToPropertyAnnotator.php:25]: The change is in response to `Visibility must be declared on all constants if your | | project supports PHP 7.1 or later`, which seems to be a PHP 7.1 feature, is there a related-change in 8.1 that affects the implementation?

Comment (JeroenDeDauw) [reply src/PropertyAnnotators/LinksToPropertyAnnotator.php:25]: Right. I confused 8.1 and 7.1

**Pull request merged**